definitions/variable/electricitysources_tags.yaml

```diff
@@ -0,0 +1,28 @@
+# This file adds needed variables for the EMPIRE data upload
+-  Secondary Energy|Electricity|{Electricity Source}:
+     description: Net electricity generation from {Electricity Source}
+     unit: EJ/yr
+- Investment|Energy Supply|Electricity|{Electricity Source}:
+    description: Investment in {Electricity Source}
+    unit: billion USD_2010/yr
+- Investment|Energy Supply|Electricity|{Common Electricity Source}:
```

Comment:
You have mismatching tags in the Variable Cost item - you use "Common Electricity Source“ in the variable name but "Electricity Source“ in the description.